### ISSUE TITLE
Call a task from another task

### ISSUE THREAD

**Issue by revolter** [pyinvoke/invoke#569]
Is it possible to call a task from another task? Simply calling the function decorated with `@task` fails with `TypeError: Task expected a Context as its first arg, got <class 'str'> instead!`.

**Comment by revolter** [comment]
So no?

**Comment by tuukkamustonen** [comment]
@revolter I know this comes in late and you probably already figured it out, but you need to pass `ctx` object along, e.g. `some_task(ctx, "your", "args")`.

**Comment by revolter** [comment]
No, I didn't figure it out, though it seems obvious now. But I was very confused after the whole migration. Thank you very much! Works like a charm.
